CHANGELOG.md

```diff
@@ -2,13 +2,15 @@
 
```

Comment:
![image](https://user-images.githubusercontent.com/55654104/114201660-bd791b80-9956-11eb-8503-73e1c9e42431.png)
I can't click to three, but I can click to two -> three. I think we should improve the user experience in this part. Let's have a meeting about how we can tackle this.